disnake/threads.py

```diff
@@ -192,7 +205,8 @@ def _from_data(self, data: ThreadPayload):
         self._type: ThreadType = try_enum(ChannelType, data["type"])  # type: ignore
         self.last_message_id = _get_as_snowflake(data, "last_message_id")
         self.slowmode_delay = data.get("rate_limit_per_user", 0)
-        self.message_count = data.get("message_count")
+        self.message_count = data.get("message_count", 0)
+        self.total_message_sent = data.get("total_message_sent", 0)
```

Comment:
```suggestion
        self.message_count = data.get("message_count") or 0
        self.total_message_sent = data.get("total_message_sent") or 0
```
Almost the same thing, but falls back to `0` if `data["message_count"]` is `None` (which shouldn't happen, but I'd rather play it safe since these are being incremented in the gateway handlers)